main.go

```
@@ -122,6 +122,7 @@ func main() {
 			os.Exit(1)
 		}
 		logger.SetOutput(logFile)
+		defer logFile.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `logFile.Close` is not checked

(errcheck)